zmon_aws_agent/aws.py

```diff
@@ -205,6 +205,8 @@ def get_running_apps(region, existing_entities=None):
 
             if (now.minute % 7) and i['InstanceId'] in existing_instances:
                 ins = existing_instances[i['InstanceId']]
+                if 'image' in ins:
```

Comment:
I think this is not needed if we will always describe the images later (without the hour limitations)